homeassistant/components/pi_hole/__init__.py

```diff
@@ -107,9 +114,12 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 @callback
 def _async_platforms(entry: ConfigEntry) -> list[Platform]:
     """Return platforms to be loaded / unloaded."""
-    platforms = [Platform.BINARY_SENSOR, Platform.UPDATE, Platform.SENSOR]
-    if not entry.data[CONF_STATISTICS_ONLY]:
-        platforms.append(Platform.SWITCH)
+    platforms = [
```

Comment:
Maybe just use a constant instead of this function?

Reply:
ahh ... you're absolutly right, this function is not needed anymore